queue is not allowed for authenticated usage build failures in 'official/$(reponame)/refs/heads/master' - '20190606.7'

@dotnet-mc-bot commented on [Thu Jun 06 2019](https://github.com/dotnet/core-eng/issues/6649)

There were a set of failures during this build. Here is a summary of these:
* https://dnceng.visualstudio.com/internal/_build/results?buildId=215255
  - **Agent:** DDVSOWINAGE321
  - **Error log:** E:\A\_work\0\s\.packages\microsoft.dotnet.helix.sdk\2.0.0-beta.19278.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(0,0): error : ArgumentException: Queue is not allowed for authenticated usage [E:\A\_work\0\s\eng\helixpubli...

@robertborr


---

@robertborr commented on [Thu Jun 06 2019](https://github.com/dotnet/core-eng/issues/6649#issuecomment-499669316)

##[error].packages\microsoft.dotnet.helix.sdk\2.0.0-beta.19278.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(51,5): error : ArgumentException: Queue is not allowed for authenticated usage
   at Microsoft.DotNet.Helix.Client.HelixApi.HandleFailedRequest(RestApiException ex) in /_/src/Microsoft.DotNet.Helix/Client/CSharp/HelixApi.cs:line 29
   at Microsoft.DotNet.Helix.Client.HelixApi.OnFailedRequest(RestApiException ex) in /_/src/Microsoft.DotNet.Helix/Client/CSharp/generated-code/HelixApi.cs:line 116
   at Microsoft.DotNet.Helix.Client.Job.<NewInternalAsync>d__10.MoveNext() in /_/src/Microsoft.DotNet.Helix/Client/CSharp/generated-code/Job.cs:line 273
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.DotNet.Helix.Client.Job.<NewAsync>d__9.MoveNext() in /_/src/Microsoft.DotNet.Helix/Client/CSharp/generated-code/Job.cs:line 190
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.DotNet.Helix.Client.HelixApi.<RetryAsync>d__69`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.DotNet.Helix.Client.JobDefinition.<SendAsync>d__81.MoveNext() in /_/src/Microsoft.DotNet.Helix/JobSender/JobDefinition.cs:line 225
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.DotNet.Helix.Sdk.SendHelixJob.<ExecuteCore>d__65.MoveNext() in /_/src/Microsoft.DotNet.Helix/Sdk/SendHelixJob.cs:line 234
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.DotNet.Helix.Sdk.HelixTask.Execute() in /_/src/Microsoft.DotNet.Helix/Sdk/HelixTask.cs:line 45
RestApiException: The response contained an invalid status code 400 Bad Request

Body: {"Message":"Queue is not allowed for authenticated usage"}



---

@robertborr commented on [Thu Jun 06 2019](https://github.com/dotnet/core-eng/issues/6649#issuecomment-499669802)

Cc: @stevenbrix

--- a/eng/helixpublish.proj
+++ b/eng/helixpublish.proj
@@ -26,9 +26,6 @@
 
     <!-- Semi-colon delimited list of commands that are executed before every test.-->
     <HelixPreCommands>$(HelixPreCommands);</HelixPreCommands>
-
-    <!-- These are the list of Windows versions that we run tests on-->
-    <HelixTargetQueues>Windows.10.Amd64.Open;Windows.7.Amd64.Open;Windows.10.Amd64.ServerRS5.Open</HelixTargetQueues>
   </PropertyGroup>
 
   <ItemGroup>

--- a/eng/pipeline.yml
+++ b/eng/pipeline.yml
@@ -50,7 +50,7 @@ jobs:
         - name: _PlatformArgs
           value: /p:Platform=$(_Platform)
         - name: _TestHelixAgentPool
-          value: 'Windows.10.Amd64.Open'
+          value: 'Windows.10.Amd64.Open%3bWindows.7.Amd64.Open%3bWindows.10.Amd64.ServerRS5.Open'
         - name: _HelixStagingDir
           value: $(BUILD.STAGINGDIRECTORY)\helix\functests
         - name: _HelixSource
@@ -96,6 +96,8 @@ jobs:
             value: '$(HelixApiAccessToken)' # from DotNet-HelixApi-Access group
           - name: _HelixCreator
             value: '' #if _HelixToken is set, Creator must be empty
+          - name: _TestHelixAgentPool
+          value: 'Windows.10.Amd64%3bWindows.7.Amd64%3bWindows.10.Amd64.ServerRS5'
       strategy:
         matrix:
           Build_Debug_x86:

--- a/eng/pipeline.yml
+++ b/eng/pipeline.yml
@@ -97,7 +97,7 @@ jobs:
           - name: _HelixCreator
             value: '' #if _HelixToken is set, Creator must be empty
           - name: _TestHelixAgentPool
-          value: 'Windows.10.Amd64%3bWindows.7.Amd64%3bWindows.10.Amd64.ServerRS5'
+            value: 'Windows.10.Amd64%3bWindows.7.Amd64%3bWindows.10.Amd64.ServerRS5'
       strategy:
         matrix:
           Build_Debug_x86:

